gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -4949,32 +4949,30 @@ subroutine GFS_physics_driver                                 &
 !       write(1000+me,*)' maxwatncb=',maxval(Stateout%gq0(1:im,k,ntlnc)),' k=',k,' kdt',kdt
 !       enddo
 
```

Comment:
Missing ## CCPP ## comment - should these be removed?  I know IPD physics driver is a mess because of these comments.

Reply:
Fortunately, this will all be gone very soon (when GFS_{physics,radiation}_driver.F90 will be removed entirely). Those comments have helped many developers a lot, therefore I strongly vote for leaving them in there for the short remaining life time of these files. Developers are not supposed to work on IPD anymore anyway, except for urgent bugfixes for GFSv16 (no other changes will be allowed in IPD once we drop the IPD-CCPP comparison tests).